### ISSUE TITLE
Watchlist-Artikel fürs Dashboard festlegen

### ISSUE THREAD

**Issue by cukabeka** [AndiLeni/statistics#90]
Ich fände toll, wenn ich bestimmte Artikel highlighten könnte. Also statt dass die komplette Statistik am Start ausgegeben wird, dass ich einige Artikel festlegen kann und verfolgen, wie die sich entwickeln. Evtl in dem Zusammenhang auch ein Feature, bei dem ich am Artikel im Edit mode in der Sidebar selbst sehen kann, wie der Artikel so performt.

Dein Vorschlag war ja:

> wenn es dir reicht, kannst du in dein einstellungen alle anderen pfade ausschließen. dein vorschlag will also genau das gegenteil davon, nämlich nur die hinterlegten pfade loggen. da muss ich mir gedanken machen wie man dies am klügsten umsetzt. sollen dann andere statistiken gar nicht mehr erfasst werden, oder soll nur die ausgabe an die gewünschten artikel angepasst werden?

Super fände ich so eine Art Watchlist, die im Fokus stehen. Ich habe ein Dashboard, in dem 7 Domains jeweils mit Aufrufen und Besuchen gezeigt werden. Auch wenn ich auf die Seitenaufrufe gehe, sind da einige Urls, die teils von Bots aufgerufen werden, oben. Mich interessieren bestimmte Seiten im vergleich untereinander aber vor allem im Zeitlichen Verlauf.

**Comment by alxndr-w** [comment]
Kannst du das an einem Beispiel visualisieren? Ich verstehe es nicht ganz.

Ich könnte mir auch gut am passenden Artikel ein Widget vorstellen. Dazu müssten aber auch evtl. Artikel-ID getrackt werden, da sich die URL ja ändern kann.

**Comment by cukabeka** [comment]
Ja, das könnte über die artikel-ID gehen, wobei das bei Datenbank Einträgen die in einem Artikel ausgegeben werden, nicht mehr funktionieren würde. 

Ich muss mal schauen, ob ich ein Beispiel finde, das das illustriert.
